bazel/test.bzl

```diff
@@ -181,10 +218,16 @@ def redpanda_cc_btest_no_seastar(
         timeout,
         srcs = [],
         defines = [],
+        cpu = 1,
+        memory = "128MiB",
         deps = []):
     native.cc_test(
         name = name,
         timeout = timeout,
+        tags = [
+            "resources:cpu:{}".format(cpu),
+            "resources:memory:{}".format(parse_bytes(memory) / 1024),
```

Comment:
did you intend to divide by `2^20`?

Reply:
Yes I thought I had a version that did that...

Reply:
Ah this is an old comment, it's divided by 2<<20 now

Reply:
Oh wait wrong one